src/Event/AuthorSubEntityEvents.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content\Event;
+
+/**
+ * Defines events for the Author sub-entities.
+ *
+ * @internal
+ */
+final class AuthorSubEntityEvents {
```

Comment:
`final class OeContentEvents {` - this class can contains another events in future, let's name it more generic.